Index: ../CYBR493A-Fall-24/In Class Activities/forTrial.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># for loop 1: lines 3-5\n# for n in range(10):\n#     # print this the value of the counter n, and then the word howdy\n#     print (\"{0}\".format(n),\"Howdy\")\n# print(\"end of the first for loop.\")\n#\n# while loop 1: lines 9-13\n\n# while n <= 10:\n#     print(\"{0}\".format(n),\"Second loop - Howdy\")\n#     n = n+1\n# print(\"end of the second for loop.\")\n# print(\"Here it is {0}\".format(n))\n\n# list and while to iterate through lines 17-23\n\nlist_of_names = [\"Erica\", \"John\", \"Hilary\", \"mj\"]\n# print(list_of_names[2])\nfind_name = \"MJ\"\nfor name in list_of_names:\n    if find_name.lower() == name.lower():\n        # print(\"found at {0} location\".format(name))\n        print(\"found\")\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../CYBR493A-Fall-24/In Class Activities/forTrial.py b/../CYBR493A-Fall-24/In Class Activities/forTrial.py
--- a/../CYBR493A-Fall-24/In Class Activities/forTrial.py	(revision 54a003d7b8808b81146644900a09a2fa63f6ce90)
+++ b/../CYBR493A-Fall-24/In Class Activities/forTrial.py	(date 1726579023584)
@@ -19,5 +19,5 @@
 find_name = "MJ"
 for name in list_of_names:
     if find_name.lower() == name.lower():
-        # print("found at {0} location".format(name))
-        print("found")
+        print("found at {0} location".format(name))
+       # print("found")
